analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -144,14 +135,16 @@ class GoMod(
     }
 
     private fun getVendorModules(projectDir: File): Set<Identifier> {
-        val vendorModulesJson = run("list", "-json", "-m", "-mod=vendor", "all", workingDir = projectDir)
+        return run(projectDir, "mod", "vendor", "-v")
             .requireSuccess()
-            .stdout
-
-        return jsonMapper
-            .readValues<ModuleDependency>(JsonFactory().createParser(vendorModulesJson))
-            .asSequence()
-            .mapTo(mutableSetOf()) { Identifier(managerName, "", it.name, it.version.orEmpty()) }
+            .stderr
+            .lineSequence()
+            .filter { it.startsWith("# ") }
+            .map {
+                val parts = it.removePrefix("# ").split(" ")
+                Identifier(managerName, "", parts[0], parts[1])
```

Comment:
How about skipping over `parts.length < 2` or `parts.length != 2`?

Reply:
This should never fail, but if it does I would actually like the analyzer to fail so that we don't silently ignore this.